test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -11,22 +12,98 @@ var destroyGraphDiv = require('../assets/destroy_graph_div');
  *
  */
 
+var PLOT_DELAY = 100;
+var MOUSE_DELAY = 20;
```

Comment:
it takes a little while for gl plots to be rendered properly even after the promise is returned.